src/diffusers/models/cross_attention.py

```diff
@@ -463,6 +465,52 @@ def __call__(self, attn: CrossAttention, hidden_states, encoder_hidden_states=No
         return hidden_states
 
 
+class Torch2AttnProcessor:
```

Comment:
Still don't like the `Torch` prefix but I'm giving up :sweat_smile: 

Torch2 is better than TorchAttention

Reply:
Oh I missed the comment for `AttnProccesor2_0`, can use that, no strong preference :)